source/docs/faq.html.md

```diff
@@ -30,7 +31,7 @@ looks just like you'd expect it to.
 
 If you would like more information the [kitchen-vagrant](https://github.com/test-kitchen/kitchen-vagrant) github page has more.
 
-### How do I add another driver other than Vagrant?
+##### How do I add another driver other than Vagrant?
 
 First, you need to make sure that the driver might already [exist](https://github.com/test-kitchen/test-kitchen/blob/master/ECOSYSTEM.md),
```

Comment:
Right now the command is kind busted - needs a help fix and also finds things it shouldn't that aren't kitchen-drivers. So yes but maybe after I fix it.